CHANGES.md

```diff
@@ -8,6 +8,7 @@
 - Fix correct behaviour for space-around and added space-evenly option
 - Fix duplicate with alt and undo only undo one step [Taiga #4746](https://tree.taiga.io/project/penpot/issue/4746)
 - Fix problem creating frames inside layout [Taiga #4844](https://tree.taiga.io/project/penpot/issue/4844)
+- Fix copy and paste very nested inside itself [Taiga #4848](https://tree.taiga.io/project/penpot/issue/4848)
```

Comment:
Yesterday we finally tagged the 1.17.2 release, could this be included for 1.17.3?